homeassistant/components/risco/const.py

```diff
@@ -1,10 +1,7 @@
 """Constants for the Risco integration."""
 
-from homeassistant.const import (
-    CONF_SCAN_INTERVAL,
-    STATE_ALARM_ARMED_AWAY,
-    STATE_ALARM_ARMED_HOME,
-)
+from homeassistant.components.alarm_control_panel import AlarmControlPanelState
```

Comment:
Side note: This state enum should preferably only be used inside the entity platform.